### DISCUSSION TITLE
Plural syntax is strange for russian language

### DISCUSSION THREAD

**Discussion by skrylnikov** [ivanhofer/typesafe-i18n#433] [General]
I want to specify only value for one, few and many. But in this syntax is impossible.
Example: 
```js
const APPLES = '{nrOfApples} {{яблок|яблоко|яблока|яблока|яблок}}'
```

**Comment by ivanhofer** [comment]
Yes, I knew that this could be strange for some languages when I implemented it.

I don't know the russian language, but if e.g no value would resolve to 'zero' you could just use an empty string for that:

```
full syntax: {{zero|one|two|few|many|other}}
your example: {{яблок|яблоко|яблока|яблока|яблок}}
just with one + few + many: {{|яблоко||яблока|яблок}}
```

Or you could try to work around this with the [`switch-case`](https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/runtime#switch-case) syntax like in this example:
https://codesandbox.io/s/typesafe-i18n-demo-forked-e66uzs?file=/index.ts:0-661

But you would need to use the same syntax for all your locales.

**Reply by skrylnikov** [reply]
Thx. It works
